Source/Core/Core/HLE/HLE.cpp

```diff
@@ -27,7 +27,7 @@ namespace HLE
 static std::map<u32, u32> s_hooked_addresses;
 
 // clang-format off
-constexpr std::array<Hook, 23> os_patches{{
+constexpr std::array<Hook, 24> os_patches{{
```

Comment:
I don't see a 24th entry being added _(regarding a HLE kprintf function to hook)_ to the array, is this an oversight?

Reply:
That's probably a merge conflict that I didn't resolve properly a long time ago. I'll see if I can dig it up.

For context I'm pretty sure I wanted to hook kprintf to get more debugging information from libogc with regards to its SD card support. I may have also needed to build libogc from source to enable additional debug logging; it's been a while.